workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -195,9 +193,16 @@ public bool TryGetEntity(long entityId, out Entity entity)
             return entityMapping.TryGetValue(entityId, out entity);
         }
 
-        public void AddGameObjectEntity(Entity entity, GameObject gameObject)
+        public bool TryGetEntityId(Entity entity, out long entityId)
         {
-            gameObjectManager.AddGameObjectEntity(entity, gameObject);
+            if (!entityManager.HasComponent<SpatialEntityId>(entity))
+            {
+                entityId = 0;
```

Comment:
If we are checking for this then <1 is more correct.

Reply:
Obsolete now as method has been removed.